test/utils.ts

```diff
@@ -0,0 +1,10 @@
+import * as sinonImport from 'sinon';
+import * as chaiImport from 'chai';
+
+declare var global: any;
+
+export const sinon = global.sinon || sinonImport;
+export const chai = global.chai || chaiImport;
```

Comment:
Can you explain the idea with the above four lines?

Reply:
This setup file will make sinon, chai, and some of their methods available for the tests in the global scope as it is run by karma and mocha before the test files. We were having difficulties getting karma specifically to pick them up.